### PR TITLE
Fixes #34692 - Add ouia-id's to all PF4 tables

### PR DESCRIPTION
**Pull request by Andrewgdewar**
#### What are the changes introduced in this pull request?

- The ticket required only a targetable ouiaId on a single table, this PR simply adds the id's to all tables, and makes 'ouiaId'  a required prop on  the Tablewrapper component.

#### What are the testing steps for this pull request?

- Open chrome dev tools. 
- Navigate to all PF4 tables in the app to ensure that the 'data-ouia-component-id' prop is now static. 

For more information on how this prop works see [here](https://www.patternfly.org/v4/developer-resources/open-ui-automation/). 

TLDR: 'ouiaId' on a component translates to a static 'data-ouia-component-id' on the dom element.

### CODE REVIEW

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailsTable.js [hunk 1]
```diff
@@ -77,6 +79,7 @@ const ContentViewVersionDetailsTable = ({
           autocompleteEndpoint,
           disableSearch,
         }}
+        ouiaId={`content_view_version_details_${snakeCase(route)}_table`}
```
Comment (Andrewgdewar) [inline webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailsTable.js:82]: This produces a static id based on the route, like this:
Screen Shot 2022-03-21 at 10 03 11 AM

File: webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js [hunk 2]
```diff
@@ -49,9 +49,9 @@ const ContentViewHistories = ({ cvId }) => {
     const taskType = task ? task.label : taskTypes[action];
 
     if (taskType === taskTypes.removal) {
-      return <>{__('Deleted from ')} <Label isTruncated key="1" color="blue" href={`/lifecycle_environments/${environment?.id}`}>{`${environment?.name ?? __('all environments')}`}</Label>{}</>;
+      return <>{__('Deleted from ')} <Label isTruncated key="1" color="blue" href={`/lifecycle_environments/${environment?.id}`}>{`${environment?.name ?? __('all environments')}`}</Label>{ }</>;
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js:52]: Was this a prettier change? I've never seen spaces being required between curly braces like this

Comment (jeremylenz) [reply webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js:52]: oh, are you using this to insert a literal space via HTML? I feel like CSS margins are a better solution in most cases

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js:52]: no this was lint, but looking at this, I think we can delete these {}.

Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js:52]: a space here does not translate down to the html. You would need to do {' '}.

Comment (jeremylenz) [reply webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js:52]: ahhh right :)

File: webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailsTable.js (hunk 1 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailsTable.js]: After changes:
Screen Shot 2022-03-29 at 9 50 41 AM

**Review by jeremylenz — APPROVED** [review]
:rocket: APJ :+1:
